ChemistryLib/PhreeqcIOData/UserPunch.h

```diff
@@ -0,0 +1,56 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include <iosfwd>
+#include <string>
+#include <vector>
+
+#include "MeshLib/PropertyVector.h"
+#include "Output.h"
+
+namespace BaseLib
+{
+class ConfigTree;
+}
+
+namespace ChemistryLib
+{
+namespace PhreeqcIOData
+{
+struct SecondaryVariable
+{
+    SecondaryVariable(std::string const name_,
```

Comment:
Either `const&` or value (or r-value). Be consistent.

Reply:
Used r-value.